app/main.py

```diff
@@ -1,3 +1,54 @@
+from functools import total_ordering
+from typing import Union
+
+
+@total_ordering
```

Comment:
the idea of this task is to do everything manually, learn what magic methods exist and how to override them, so let's implement them without this decorator ;)